ir/src/symbol_table/symbol_binding.rs

```diff
@@ -23,13 +23,13 @@ pub(crate) enum SymbolBinding {
 impl Display for SymbolBinding {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Self::Constant(_) => write!(f, "Constant"),
-            Self::Trace(binding) => {
-                write!(f, "Trace in segment {}", binding.trace_segment())
+            Self::Constant(_) => write!(f, "ConstantBinding"),
+            Self::Trace(_) => {
+                write!(f, "TraceBinding")
             }
             Self::PublicInput(_) => write!(f, "PublicInput"),
             Self::PeriodicColumn(_, _) => write!(f, "PeriodicColumn"),
-            Self::Variable(_) => write!(f, "Variable"),
+            Self::Variable(_) => write!(f, "VariableBinding"),
             Self::RandomValues(_, _) => write!(f, "RandomValues"),
```

Comment:
Question: is there a reason why we sometimes add "Binding" at the end and sometimes don't?

Reply:
I think it's clearer in the errors if it has binding in it, but we don't ever refer to the other ones as bindings (even though they are).